doc/source/frequently_asked_questions.rst

```diff
@@ -88,7 +88,7 @@ GitHub Actions
 How do I set different Python versions for GitHub CI?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The default is Python 3.13 for ``_tests-on-pr.yml`` and ``_publish-docs-on-release.yml``. Python 3.11, 3.12, and 3.13 are used for ``_matrix-and-codecov-on-merge-to-main.yml``. To override the default, modify the three ``.yml`` files above in ``.github/workflows/`` as shown below:
+The default Python version is |PYTHON_DEFAULT_MAX_VERSION| for ``_tests-on-pr.yml`` and ``_publish-docs-on-release.yml``. Python |PYTHON_DEFAULT_MIN_VERSION| through |PYTHON_DEFAULT_MAX_VERSION| are for ``_matrix-and-codecov-on-merge-to-main.yml``. To override the default, modify the three ``.yml`` files above in ``.github/workflows/`` as shown below:
```

Comment:
ah, this may answer my question...but it is still not 100% clear to me actually what is going on here.